EXAMPLES.md

```diff
@@ -162,6 +162,23 @@ end
 In this case the `recursive: true` option mirrors the same options which are
 available to [`Net::{SCP,SFTP}`](http://net-ssh.github.io/net-scp/).
```

Comment:
confirmed this older documentation about `recursive: true` is still true with SFTP enabled.